backend/main.py

```
@@ -10,11 +10,19 @@
 from fastapi import FastAPI, HTTPException, Request
 from fastapi.middleware.cors import CORSMiddleware
 from fastapi.responses import Response
+from slowapi import _rate_limit_exceeded_handler
+from slowapi.errors import RateLimitExceeded
+from slowapi.middleware import SlowAPIMiddleware
+from utils.limiter import Limiter, get_remote_address, limiter
+
+limiter = Limiter(key_func=get_remote_address)
```

Ruff (0.12.2)

Redefinition of unused `limiter` from line 16

(F811)